cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -65,9 +65,13 @@ def export_model_predictions(cloud_event: http.CloudEvent) -> None:
         chunks_ref,
     )
 
-    # Set the error message to the Series' json string for testing
-    # purposes. In the actual implementation, the Series should be stored somewhere.
-    raise NotImplementedError(h3_predictions.to_json())
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(
+        f"{prediction_type}/{model_id}/{study_area_name}/{scenario_id}/{chunk_id}.csv"
+    )
+    with blob.open("w+") as fd:
+        h3_predictions.to_csv(fd)
```

Comment:
was this schema confirmed with Jesus?

Reply:
Yes. The final schema (after merging) will be h3_index,scenario_0_id,scenario_1_id, etc. The schema for this intermediary step is just h3_index,prediction.